src/components/MyTimeComponents/Reschedule/Reschedule.scss

```diff
@@ -1,108 +1,60 @@
-.rescheduleModal {
-  hr {
-    margin: 0;
-    color: #d9d9d9;
-    opacity: 1;
-  }
-
+.reschedule-modal {
   .ant-modal-content {
     padding: 0 0 24px 0 !important;
     border: none !important;
     border-radius: 28px !important;
-
-    .selected {
-      border: none;
-      background: var(--bottom-nav-color);
-      box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
-    }
-    .booked {
-      background: var(--bottom-nav-color);
-    }
   }
 
-  .list {
+  .reschedule-options {
+    align-self: center;
     margin: 0 16px;
-    padding: 8px 8px 16px 8px;
-    border-left: 0;
-    border-right: 0;
+    padding: 12px 8px 16px 8px;
+    margin-top: 20px;
     flex-wrap: nowrap;
     overflow-x: scroll;
     display: flex;
-    gap: 20px;
 
-    .res-card {
-      border-radius: 8px;
-      background: transparent;
-      border: 1px solid #d9d9d9;
-      padding: 10px 15px;
-      display: flex;
-      flex-direction: column;
-      gap: 10px;
-      text-align: center;
-      min-width: 75px;
-      align-items: center;
+    .ant-radio-button-wrapper {
+      margin: 2px;
+      border-inline-start-width: 1px;
+      border-radius: 10px;
     }
 
-    p {
-      margin: 0;
-      font-size: 1.286em;
-      font-weight: 500;
-      white-space: nowrap;
+    .ant-radio-button-wrapper:not(:first-child)::before {
+      display: none;
     }
 
-    .res-card > p:last-child {
-      font-weight: 500;
-    }
-
-    .res-card.selected > p:last-child {
-      font-weight: 700;
-      color: var(--bottom-nav-text-color);
-    }
-  }
-
-  .slot-batch {
-    display: flex;
-    flex-direction: column;
-    gap: 10px;
-
-    div {
-      display: flex;
-      gap: 10px;
+    .ant-radio-group-solid .ant-radio-button-wrapper-checked:not(.ant-radio-button-wrapper-disabled) {
```

Comment:
instead of targeting this, go for this ".ant-radio-button-wrapper-checked""

Reply:
> no need for a reschedule button - effectuate the postpone as soon as you press one of the options

as @tijlleenders mentioned to reschedule as soon as user presses the option, we don't need checked based styles anymore